usaspending_api/idvs/v2/urls_idvs.py

```diff
@@ -14,4 +15,5 @@
     url(r"^awards/$", IDVAwardsViewSet.as_view()),
     url(r"^funding/$", IDVFundingViewSet.as_view()),
     url(r"^funding_rollup/$", IDVFundingRollupViewSet.as_view()),
+    url(r"^count/federal_account/(?P<requested_award>[A-Za-z0-9_. -]+)/$", IDVFederalAccountCountViewSet.as_view()),
```

Comment:
If this is a `get` request, does the optional `piid` param need to be defined here?

Reply:
I wasn't sure if the pattern was to include params in the url definition or not. If the code is functional and aligns with existing endpoints then no changes are requested.